src/components/error-list/error-list-item.component.tsx

```diff
@@ -111,7 +111,6 @@ export default function ErrorListItem({
   );
 }
 
-// TODO: Simplify that
```

Comment:
@pectom 
Szczerze mówiąc nie mam pomysłu jak.
To raczej nie przeszkadza i nie myślę, że warto poświęcać na to czas, więc usunąłem